examples/llm/run/llama_pretraining.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+from functools import partial
```

Comment:
Add 1-2 lines explaining what this example contains and the pre-reqs. 
1. NeMo-Run from https://github.com/NVIDIA/NeMo-Run
2. Optional slurm, otherwise can be run locally